File: tools/source/sdks/cs-sdk/examples/TaskPullOnceExample.cs
using System;
using GameAgentEngine.SDK;

var client = new GameAgentEngineClient(Environment.GetEnvironmentVariable("GAE_SERVER") ?? "http://127.0.0.1:8080", Environment.GetEnvironmentVariable("GAE_KEY") ?? "dev-key");
Console.WriteLine(await client.ListPendingRuntimeTasksAsync(Environment.GetEnvironmentVariable("GAE_CONSUMER") ?? "game_client", 1));

File: tools/source/sdks/java-sdk/examples/TaskPullOnceExample.java
public class TaskPullOnceExample {
    public static void main(String[] args) throws Exception {
        var client = new GameAgentEngineClient(
            System.getenv().getOrDefault("GAE_SERVER", "http://127.0.0.1:8080"),
            System.getenv().getOrDefault("GAE_KEY", "dev-key")
        );
        var consumer = System.getenv().getOrDefault("GAE_CONSUMER", "game_client");
        System.out.println(client.listPendingRuntimeTasks(consumer, 1));
    }
}

